Update Metadata + Screenshots

Ashnidh opened the pull request:
I have updated the metadata and added 3 screenshots.

Ashnidh commented:
Sir, as per your suggestions I have updated the metadata. Thank you.

Ashnidh commented:
Thank you. Done.

quozl commented:
Thanks.  Reviewed.

* [ ] please don't change the order of metadata, as it makes it much harder to use `git blame` and other tools to see when the metadata was added, and the order has no impact on function,
* [ ] please rewrite your patches for the separate changes; I don't like how the commits have combined both superficial metadata changes _and_ screenshots.

What was wrong with the existing screenshots?

Ashnidh commented:
> Thanks. Reviewed.
> 
> * [ ]  please don't change the order of metadata, as it makes it much harder to use `git blame` and other tools to see when the metadata was added, and the order has no impact on function,
> * [ ]  please rewrite your patches for the separate changes; I don't like how the commits have combined both superficial metadata changes _and_ screenshots.
> 
> What was wrong with the existing screenshots?

Sir, 
I have restored the order of metadata. 
I have done it for a gci task to update metadata and screenshots as well.
I have used a virtual machine and set the display resolution of the virtual window to 1024x768 to capture the new screenshots. I have replaced them. 
I have added a couple of screenshots (in default bg color) to the existing ones
Regards.

Ashnidh commented:
Thanks. Reviewed. 
Regards.
